pkg/private/zip/zip.bzl

```diff
@@ -39,6 +37,7 @@ def _pkg_zip_impl(ctx):
     args.add("-t", ctx.attr.timestamp)
     args.add("-m", ctx.attr.mode)
     inputs = []
+    file_deps = []
```

Comment:
Perhaps declare this adjacent to where it is used, perhaps on line 49?

Reply:
Moved.